api/v01/entities/vrp_result.rb

```diff
@@ -56,71 +53,78 @@ class VRPResultDetailedCosts < Grape::Entity
+      expose :begin_time, documentation: { type: Integer, desc: 'Time visit starts' }
+      expose :end_time, documentation: { type: Integer, desc: 'Time visit ends' }
       expose :departure_time, documentation: { type: Integer, desc: '' }
-      expose :service_id, documentation: { type: String, desc: '' }
-      expose :pickup_shipment_id, documentation: { type: String, desc: '' }
-      expose :delivery_shipment_id, documentation: { type: String, desc: '' }
+      expose :service_id, documentation: { type: String, desc: 'Internal reference of the service' }
+      expose :pickup_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
+      expose :delivery_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
       expose :detail, using: VrpResultSolutionRouteActivityDetails, documentation: { desc: '' }
+      expose :type, documentation: { type: String, desc: 'store, rest, service, pickup or delivery' }
+      expose :current_distance, documentation: { type: Integer, desc: 'Travel distance from route start to current point' }
+      expose :alternative, documentation: { type: Integer, desc: 'When one service has alternative activities, index of the chosen one' }
     end
 
     class VrpResultSolutionRoute < Grape::Entity
-      expose :vehicle_id, documentation: { type: String, desc: 'Reference of the vehicule used for the current route' }
+      expose :vehicle_id, documentation: { type: String, desc: 'Internal reference of the vehicule used for the current route' }
       expose :activities, using: VrpResultSolutionRouteActivities, documentation: { is_array: true, desc: 'Every step of the route' }
+      expose :total_travel_time, documentation: { type: Integer, desc: 'Sum of every travel time within the route' }
       expose :total_distance, documentation: { type: Integer, desc: 'Sum of every distance within the route' }
       expose :total_time, documentation: { type: Integer, desc: 'Sum of every travel time and activity duration of the route' }
+      expose :total_waiting_time, documentation: { type: Integer, desc: 'Sum of every idle time within the route' }
       expose :start_time, documentation: { type: Integer, desc: 'Give the actual start time of the current route if provided by the solve' }
       expose :end_time, documentation: { type: Integer, desc: 'Give the actual end time of the current route if provided by the solver' }
       expose :geometry, documentation: { type: String, desc: 'Contains the geometry of the route, if asked in first place' }
       expose :initial_loads, using: VrpResultDetailQuantities, documentation: { is_array: true, desc: 'Give the actual initial loads of the route' }
       expose :costs, using: VRPResultDetailedCosts, documentation: { desc: 'The impact of the current route within the solution cost' }
     end
 
-    class VrpResultSolutionUnassigneds < Grape::Entity
+    class VrpResultSolutionUnassigned < Grape::Entity
```

Comment:
Changing an object name which is exposed, is always akward as it could be used. Nevertheless, as the documentation is badly generated, and these objects are the unassigned ones, it seems quite ok.